### ISSUE TITLE
BTW Don't Use Enable all permissions

### ISSUE THREAD

**Issue by FlipMods** [Sapphire-Inc/Sapphire#2]
_Enable all permissions_ functionality just bans you

**Comment by uhidontkno** [comment]
> _Enable all permissions_ functionality just bans you

If you weren't a fucking illiterate using chatgpt instead of your brain and ACTUALLY **READ**, you would know that i-ready cheats have a chance of getting you banned. Go eat sand you motherfucker.

**Comment by SapphirePlaceholder** [comment]
> _Enable all permissions_ functionality just bans you

this is false, i-ready does not have client side detection detecting if you altered user privileges and you cannot open a lesson when you enable all permissions because it enables new lessons which is the only way it could detect.
